ui/packages/atlasmap-core/src/services/file-management.service.ts

```diff
@@ -160,25 +160,23 @@ export class FileManagementService {
   }
 
   /**
-   * Delete mapping files on the runtime.
+   * Delete mapping project on the runtime.
    */
   resetMappings(): Promise<boolean> {
     return new Promise<boolean>((resolve) => {
-      const url = this.cfg.initCfg.baseMappingServiceUrl + 'mapping/RESET';
-      this.cfg.logger!.debug('Reset Mappings Request');
+      const url = this.cfg.initCfg.baseAtlasServiceUrl + 'project/0';
```

Comment:
Is reset really always delete in this context?

Reply:
It looks like so, the backend just deletes the directory